packages/client/src/pages/Dashboard/Dashboard.page.tsx

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
+import { AppHeader } from '../../components/AppHeader/AppHeader';
 import { Page } from '../../components/Page/Page';
 import { LeaderboardWidget } from '../../components/LeaderBoardWidget/LeaderBoardWidget';
 
 export const DashboardPage = () => (
   <Page title="Dashboard" type="dashboard">
+    <AppHeader />
```

Comment:
Can we actually put this in the `Page` element? Just thinking about it now...a lot of places will use it. Will be nice to include it automatically I think